fastlane/docs/Platforms.md

```diff
@@ -1,6 +1,6 @@
 # Support for multiple platforms
 
-`fastlane` can handle multiple operating systems in one `Fastfile`. This is super useful for projects supporting both iPhone and Mac OS apps.
+`fastlane` can handle multiple operating systems in one `Fastfile`. This is super useful for projects supporting iPhone, Mac OS X and Android apps.
```

Comment:
I think it might be better to replace 'operating systems' -> 'app platforms'. That might prevent confusion about 'operating systems' meaning Windows/Mac/Linux